pages/learn/develop/container-contracts.md

```diff
@@ -54,7 +54,7 @@ requires:
   - type: "sw.supervisor"
     version: ">=10.6.17"
   - type: "sw.l4t"
-    version: "32.2"
+    version: ">=32.2"
```

Comment:
So currently the contracts engine must detect this to be a semver for it to do a semver check, so in this case it would actually have to be 
```suggestion
    version: ">=32.2.0"
```

Reply:
Ah, got it thanks 👍